docs/en/stack/ml/nlp/ml-nlp-elser.asciidoc

```diff
@@ -385,6 +385,35 @@ The following sections provide information about how ELSER performs on different
 hardwares and compares the model performance to {es} BM25 and other strong 
 baselines such as Splade or OpenAI.
 
+[discrete]
+[[version-overview]]
+=== Version overview
+
+The ELSER model has two main versions: ELSER V2 and ELSER V1.
+ELSER V2 has a **platform specific** version that is designed to run on Linux 
+with an x86-64 CPU architecture and a **platform agnostic or portable** version 
+that can be run on any platform.
+
+[discrete]
+==== ELSER V2
+
+The biggest change in ELSER V2 is the introduction of the first platform 
+specific ELSER model - that is, a model that can only run on Linux with an 
+x86-64 CPU architecture. The platform specific model is designed to work best on 
+newer Intel CPUs, but it works on AMD CPUs as well. It is recommended to use the 
+new platform specific Linux-x86-64 model for all new users of ELSER as it is 
+significantly faster than the platform agnostic (portable) model which can be 
+run on any platform. ELSER V2 produces significantly higher quality embeddings 
+than ELSER V1. Regardless of which ELSER V2 model you use (platform specific or 
+platform agnostic (portable)), the particular embeddings produced are the same.
+
+[discrete]
+==== ELSER V1
+
+The initial version of the ELSER model was introduced in 8.9. However, ELSER V1 
+in {current} is more performant than the initial version due to the {es} 
```

Comment:
Because these models have multiple types of performance (inference throughput, inference and query latency, and embedding/semantic search quality), we should be clear that the only metric that we know has improved is throughput. Thus, we could word this as:

"However, ELSER V1 
in {current} has higher throughput than the initial version due to {es} platform improvements"

Reply:
Section has been deleted based on our conversation earlier.